common/components/controllers/MainController.jsx

```diff
@@ -24,6 +24,35 @@ class MainController extends React.Component<{||}, State> {
     };
   }
 
+  static iframeresizerInParent(...args){
+    // if we are running within an iframe, and the parent is running IframeResizer then do this
+      if(!args[0].data.includes('[iFrameSizer]')) return
+      // let the height go auto rather than being the height defined by the parent - since we will be resizing
+      const style = document.createElement("style")
+      style.textContent = ".Profile-primary-container.frame-full .AboutGroup-card-container .row .ProjectCardContainer .row {height: auto!important;}"
+      document.head.appendChild(style)
+      if(!window.iFrameResizer) window.iFrameResizer={}
+      window.iFrameResizer.inParent=true
+      let func
+      while((func=window.iFrameResizer.onInParent?.shift()))
```

Comment:
What is this line doing?

Reply:
I will add a comment to the code.  There could be multiple components needing to be re-rendered after iframe resizer starts, so onInParent is an array of callbacks from those components to run after iframeresizer starts.  Components may need to rerender after iframeresizer starts because their style needs to change if rendering in an iframe that resizes v. rendering in an iframe that doesn't resize.